<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QDialog" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(198, 224, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>172</width>
     <height>323</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="NumeLabel">
      <property name="text">
       <string>FirstName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FirstnameText"/>
    </item>
    <item>
     <widget class="QLabel" name="PrenumeLabel">
      <property name="text">
       <string>LastName</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LastnameText"/>
    </item>
    <item>
     <widget class="QLabel" name="EmailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EmailText"/>
    </item>
    <item>
     <widget class="QLabel" name="UsernameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UsernameText"/>
    </item>
    <item>
     <widget class="QLabel" name="PasswordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordText">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RegisterButton">
      <property name="text">
       <string>REGISTER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CloseButton">
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
